src/Buttons.js

```diff
@@ -5,7 +5,7 @@ import './App';
 function Buttons() {
 
```

Comment:
on line 16 and 17 where you're using emojis [here](https://github.com/MelanieRogoff/SpotiMood/blob/a86647e7fc7c17a14421a4c03c1ae788e533455b/src/Buttons.js#L15)

1. **Browser Backwards compatibility:** this is great and works fine. but we typically don't want to do this because emoji text uses encoding, and it's possible that on some browser version, these encoding do not exists. so your emoji text wont show up.
    - instead of using the DOM base elements like `<button>` and spending time customizing and styling that to look good. we should just use a 3rd party UI library. a super popular one is [Material-UI](https://material-ui.com/). That way we don't reinvent the wheel and can focus on the important stuff in our apps. (Plus the UI will look professional by default with little to no work)
    - **NOTE**: dont do that update on this PR. let's merge what we have and iterate on it on the next PR.

2. **Naming Convention practice:**  usually when you create a function/component, you want the name to be able to signify what the function/component is actually doing. since you made a `Button` component, as a consumer I expect that when I call `<Buttons/>` a button will appear, but it does not.
    - try to see if you can rename the component something more informative.
    - here's a good reference to skim through https://www.freecodecamp.org/news/javascript-naming-conventions-dos-and-don-ts-99c0e2fdd78a/

Reply:
Did not change per your request. Changed name of Buttons.js & <Buttons> to be SpotifyPlaylistLaunchers.js & <SpotifyPlaylistLaunchers>, respectively, which is in the latest commit.